source-code/sdk-js/src/adapter-sveltekit/ast-transformation/vite-plugin.ts

```diff
@@ -146,10 +145,13 @@ const moveExistingRoutesIntoSubfolder = async (config: TransformConfig) =>
 
 let viteServer: ViteDevServer | undefined
 
-export const unplugin = createUnplugin(() => {
+export const plugin = () => {
```

Comment:
we needed to get rid of `unplugin` because `esbuild` can't bundle the output due to some `webpack` dependency. As `SvelteKit` requires `vite` to function, we can ignore all other `plugins` and just release a `vite` plugin.